desktop/apps/consignments2/components/upload_photo/index.js

```diff
@@ -22,7 +22,9 @@ function UploadPhoto (props) {
   } = props
   const b = block('consignments2-submission-upload-photo')
 
-  const nextEnabled = skipPhotoSubmission || (!skipPhotoSubmission && uploadedImages.length > 0 && processingImages.length === 0)
+  const imagesInProgress = uploadedImages.length > 0 && processingImages.length > 0
```

Comment:
Your version is more informative at a glance, but could also do `uploadedImages.length` since `0` is falsy